routes/index.js

```diff
@@ -35,14 +38,56 @@ const routes = {
 
 // Bind Routes
 exports = module.exports = (app) => {
-  app.get('/', routes.views.index);
-  app.get('/directors', routes.views.list_directors);
-  app.get('/themes', routes.views.list_themes);
-  app.get('/genres', routes.views.list_genres);
-  app.get('/movies', routes.views.list_movies);
-  app.get('/themes/:name', routes.views.theme_profile);
-  app.get('/genres/:name', routes.views.genre_profile);
-  app.get('/directors/:name', routes.views.director_profile);
-  app.get('/movies/:name', routes.views.movie_profile);
-  app.get('/contact', routes.views.contact);
+
+  // Use express's router as middleware
+  // Base routes are accessible with /{locale}/ prepended
+  const viewRouter = require('express').Router();
+
+  viewRouter.use((req, res, next) => {
+    // Extract locale (e.g.) 'en' or 'chn' from full url
+    const locale = req.originalUrl.split('/')[1];
+
+    // If /en/ or /chn/ not included in url
+    if ([ 'en', 'chn', ].includes(locale) != true) {
```

Comment:
Please use strict equality only.

If you you need to coerce types, do it explicitly.